releng/com.b2international.snowowl.server.update/assembly/common/configuration/tomcat-server.xml

```diff
@@ -15,43 +15,62 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<Server>
+<Server port="-1">
   <!--APR library loader. Documentation at /docs/apr.html -->
   <Listener className="org.apache.catalina.core.AprLifecycleListener" SSLEngine="on" />
   <!--Initialize Jasper prior to webapps are loaded. Documentation at /docs/jasper-howto.html -->
-  <Listener className="org.apache.catalina.core.JasperListener" />
   <!-- Prevent memory leaks due to use of particular java/javax APIs-->
   <Listener className="org.apache.catalina.core.JreMemoryLeakPreventionListener" />
   <Listener className="org.apache.catalina.mbeans.GlobalResourcesLifecycleListener" />
   <Listener className="org.apache.catalina.core.ThreadLocalLeakPreventionListener" />
   <Listener className="org.eclipse.virgo.web.tomcat.support.ServerLifecycleLoggingListener"/>
+  <Listener className="org.apache.catalina.security.SecurityListener" />
 
   <Service name="Catalina">
-    <Connector port="8080" protocol="HTTP/1.1"
-               connectionTimeout="20000"
-			   asyncTimeout="60000"
-               redirectPort="8443" />
+    <Connector address="127.0.0.1" port="8080" 
```

Comment:
This will make the non-HTTPS port listen on the local interface only; please change the `address` value to `0.0.0.0`.